src/app/contexts/Migration/Migration.tsx

```diff
@@ -83,9 +84,10 @@ interface Properties {
 
 const MigrationContext = React.createContext<any>(undefined);
 
-const ONE_SECOND = 1000;
+const MIGRATION_LOAD_INTERVAL = 5000;
+const INITIAL_MIGRATION_LOAD_TIME = 1000;
```

Comment:
shouldn't need this if you make the initial load request the status and only apply an interval afterwards? So you never rely on the interval for the initial check, only when there are pending transaction you add one to occasionally check if the status has changed